crates/proof-of-sql/src/proof_primitive/hyrax/base/hyrax_helpers.rs

```diff
@@ -0,0 +1,187 @@
+//! This module duplicates a lot of code that is in dynamic dory. Ideally this all will be cenralized.
```

Comment:
Understood. I think it would be ok to save this for a future PR. 

NIT: I like to add a 

# Split

Section to my PRs that contain items that will be addressed later on